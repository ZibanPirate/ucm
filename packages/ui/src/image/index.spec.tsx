import { render } from "@testing-library/react";

import { Image } from ".";

describe(`Testing component '${Image.name}' :`, () => {
  it(`should render with image url '/assets/test.svg'`, () => {
    const { container } = render(<Image url="/assets/test.svg" />);
    expect(container).toMatchSnapshot();
  });

  it(`should render an auto-sized '${Image.name}' component image url '/assets/test.svg'`, async () => {
    const { container } = render(<Image url="/assets/test.svg" size="auto" />);
    expect(container).toMatchSnapshot();
  });
});
