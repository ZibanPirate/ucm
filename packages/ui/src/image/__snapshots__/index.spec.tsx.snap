// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component 'Image' : should render an auto-sized 'Image' component image url '/assets/test.svg' 1`] = `
<div>
  <img
    src="/assets/test.svg"
    style="object-fit: cover; width: 100%;"
  />
</div>
`;

exports[`Testing component 'Image' : should render with image url '/assets/test.svg' 1`] = `
<div>
  <img
    src="/assets/test.svg"
    style="width: 200px; height: 200px; object-fit: cover;"
  />
</div>
`;
